src/index.ts

```diff
@@ -17,3 +17,6 @@ export {Workspace, WorkspaceRepo} from './workspace';
 
 /** Expose all operations. */
 export * from './workspace-ops';
+
+/** Expose command operations. */
+export * from './util/exec';
```

Comment:
Exported to use in `polymer-modulizer` as well. Can remove if this isn't preferred or the correct way of doing so.

Reply:
Was thinking about it, maybe it would be better to move the git status check in here and then expose the git commands? or both expose git cmds and exec?

Reply:
I definitely get the reasoning to expose this for modulizer, code reuse is a good thing. But it's also not really this libraries job/focus, it's more of an internal helper. 

I think it's fine that we re-implement something similar in polymer-modulizer. If we ever need to use it in a 3rd library, I'd rather see it pulled into a small helper package all of its own.

wdyt?

Reply:
I could definitely see pulling it out into a small helper package. 👍 I may do that after the fact and just remove the export on this package for now.